### ISSUE TITLE
Cannot install because of invalid syntax

### ISSUE THREAD

**Issue by benjaminoakes** [havoc-io/homebrew-mutagen#2]
```
$ brew install mutagen-io/mutagen/mutagen
==> Tapping mutagen-io/mutagen
Cloning into '/usr/local/Homebrew/Library/Taps/mutagen-io/homebrew-mutagen'...
remote: Enumerating objects: 29, done.
remote: Counting objects: 100% (29/29), done.
remote: Compressing objects: 100% (22/22), done.
remote: Total 178 (delta 8), reused 27 (delta 7), pack-reused 149
Receiving objects: 100% (178/178), 27.11 KiB | 5.42 MiB/s, done.
Resolving deltas: 100% (45/45), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/mutagen-io/homebrew-mutagen/Formula/mutagen@0.10.rb
No available formula with the name "mutagen@0.10"
In formula file: /usr/local/Homebrew/Library/Taps/mutagen-io/homebrew-mutagen/Formula/mutagen@0.10.rb
Expected to find class MutagenAT010, but only found: Mutagen.
Error: Cannot tap mutagen-io/mutagen: invalid syntax in tap!
```

I see messages about v0.11 on GitHub, so maybe some version was not bumped?

**Comment by xenoscopic** [comment]
Thanks for reporting this!  It looks like I forgot to rename the formula class when creating the @10 formula.  For some reason Homebrew didn't print a warning when I did the test install.  Can you do a `brew update` and try again?

**Comment by benjaminoakes** [comment]
Looks like it worked 👍 
Thanks for the quick fix!

```
# ...
==> Summary
🍺  /usr/local/Cellar/mutagen/0.11.0: 5 files, 78.0MB, built in 20 seconds
$ mutagen
Mutagen is a remote development tool built on high-performance synchronization

Usage:
  mutagen [flags]
  mutagen [command]

# ...
```

**Comment by xenoscopic** [comment]
Cool, thanks for the heads up!

**Comment by kimaegaii** [comment]
@havoc-io 
Hi I'm having the same issue. I get this back... 

```
SyntaxError: invalid syntax
>>> /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
  File "<stdin>", line 1
    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
    ^
```
I copy pasted into my macOS terminal from the brew website and haven't been able to found any solutions. (I am also a big time nube just trying to get python 3 working because the mac I have shipped with 2.7 I guess.)

Thank you

**Comment by xenoscopic** [comment]
@kimaegaii This looks like an issue with the Homebrew installation and not Mutagen.  It looks like you're trying to run a shell command inside of a Python shell.  You'll need to run it in a system shell (e.g. Bash or zsh).
